meta-balena-common/recipes-containers/balena/balena/balena-healthcheck-image-load

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+
+until $(systemctl is-active --quiet balena.service) ; do
+        sleep 1
+done
+
+balena load -i /usr/lib/balena/balena-healthcheck-image.tar
+
+# The healthcheck image was previously called hello-world.
+# Clean it up
+balena rmi hello-world 2> /dev/null || true
```

Comment:
If the hello-world image does not exist (already removed) the last command will return failed and the script result will be failed. Maybe that's why it needs to be executed in the background below.
You could try to perform the rmi before the "balena load" so that the script result will only fail if the load actually fails.

Reply:
until `ExecStartPost` exits, systemd never marks the service as active.
Hence the `until` loop inside the script keeps waiting